cmd/build/v2/config.go

```diff
@@ -14,12 +14,20 @@
 package v2
 
 import (
+	"os"
+	"strconv"
+
 	oktetoLog "github.com/okteto/okteto/pkg/log"
 	"github.com/okteto/okteto/pkg/model"
 	"github.com/okteto/okteto/pkg/okteto"
 	"github.com/spf13/afero"
 )
 
+const (
+	// OktetoEnableSmartBuilds key representing whether the feature flag to enable smart builds is enabled or not
+	OktetoEnableSmartBuilds = "OKTETO_SMART_BUILDS_ENABLED"
```

Comment:
Could we add the EnvVar suffix to make it clear the intention of the constant?
```suggestion
	OktetoEnableSmartBuildEnvVar = "OKTETO_SMART_BUILDS_ENABLED"
```